cita-auth/src/handler.rs

```diff
@@ -377,7 +377,6 @@ impl MsgHandler {
         // we will send req for all height
         // so don't too frequent
         if check && self.history_heights.is_too_frequent() {
-            warn!("Too frequent to send request!");
```

Comment:
Why delete it?
info or debug or trace?

Reply:
This log is meaningless. Even if the program enters this logic, it does not mean `Too frequent to send request.`

Reply:
If `is_too_frequent` is useless, we should create another PR to fix it.

Reply:
But this also great influence on this PR. Because this log print too frequent, they will consume a lot of cpu time, resulting in slower auth execution.
```
2019-03-14T02:21:06.822011671+08:00 - INFO - Session initialized
2019-03-14T02:21:08.063911004+08:00 - WARN - Too frequent to send request!
2019-03-14T02:21:08.141921838+08:00 - WARN - Too frequent to send request!
2019-03-14T02:21:08.142061364+08:00 - WARN - Too frequent to send request!
2019-03-14T02:21:08.169812155+08:00 - WARN - Too frequent to send request!
2019-03-14T02:21:08.169926582+08:00 - WARN - Too frequent to send request!
2019-03-14T02:21:08.198717953+08:00 - WARN - Too frequent to send request!
2019-03-14T02:21:08.198803486+08:00 - WARN - Too frequent to send request!
2019-03-14T02:21:08.229030371+08:00 - WARN - Too frequent to send request!
```
So I think is better to handle this issue in this PR.